### ISSUE TITLE
Game Crashing and Disconnecting

### ISSUE THREAD

**Issue by Karawasa** [Karawasa/Element-TD#488]
I've gotten some direct messages but also there are posts and comments about this;

At wave 43, the server drops me. Then it doesn't connect. It happens every time, and has happened to me 4 times now.

why i keep dc around round 29-34 everytime auto dc and cant reconnect to the server

keep stuck at stage 34 fuck bug

my game is crashing at level 37
how can i fix it?

after wave 36 and came to wave 37, dota 2 crash, when I try to reconnect , I already lost.

Can someone look into this? Thanks!

**Comment by crsaracco** [comment]
It's crashing some time during the loading screen for me (on Arch linux, if it matters).

Dota2 Version 7.15, but has been happening for a day or two now.

Edit: seems to work fine on Windows, though.

**Comment by ckhung5** [comment]
Yes.. Same for me as the dota 2 client crashed while loading Element td . I am using Mac but windows seems fine.

**Comment by ynohtna92** [comment]
@crsaracco @ckhung5 Is this still happening? As far as I can tell this is probably a valve issue.

**Comment by ckhung5** [comment]
hey @ynohtna92 . Mine is working right now. I just change the graphic settings and restart my Mac.
